tests/e2e-tests/config/fixture-loaders.js

```diff
@@ -0,0 +1,297 @@
+/* eslint-disable no-console */
+/**
+ * External dependencies
+ */
+const WooCommerceRestApi = require( '@woocommerce/woocommerce-rest-api' )
+	.default;
+
+require( 'dotenv' ).config();
```

Comment:
Environment variables are not available this early so we load them.

Reply:
I think we can, I was testing using node directly, which doesn’t have import, jest have babel support, but I'm not sure if it is enabled on globalSetup and globalTeardown, will test.